src/locale/ro.js

```diff
@@ -19,7 +19,7 @@ const locale = {
   },
   relativeTime: {
     future: 'peste %s',
-    past: 'acum %s',
+    past: 'în urmă cu %s',
```

Comment:
can we use `%s în urmă`

I see this on https://github.com/moment/moment/blob/develop/src/locale/ro.js#L55

Reply:
No, that's worse. My version is the correct one.
I'll make a pull request to the moment library as well.

Reply:
Thanks, I'll mark this PR for more code review by native speakers before it gets merged.

Reply:
Good plan 👍